apps/nextjs-website/src/components/organisms/Layout/useScrollUp.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable functional/no-expression-statements */
+import { useRef, useState, useEffect } from 'react';
+
+export const useScrollUp = () => {
+  const prevScrollY = useRef(0);
+  const [scrollUp, setScrollUp] = useState(true);
+
+  useEffect(() => {
+    const updateVisibility = () => {
+      const scrollY = window.scrollY;
+
+      if (Math.abs(scrollY - prevScrollY.current) >= 100) {
```

Comment:
it is better to avoid magic number and have a costant outsided function with this value like:
```
const SCROLL_UP_TRESHOLD=100;
```
This way the code is also more understandable.